src/apps/content/actions/product-actions.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Dependencies.
```

Comment:
In the web app we use the top lines to explain in 1 sentence what the component is supposed to do

in most cases it is obvious but still helps a bit to get into the context of the application

eg

```
# This component lazy loads images (duh…)
#
# Consists of two components
#   - The container that does the actual lazyloading
#   - The elements send events to notify the container of their existence
#
#
# Usage:
#   <body data-component="LazyLoadContainer">
#     <img data-component="LazyLoadElement" data-lazyload-url="/images/foobar.png"/>
#   </body>
#
```

or
```
# Component displaying prerendered content into a fullscreen overlay
#
# Usage:
#
#   <div data-component="FullscreenOverlayContent">
#      ...
#   </div>
#
```